src/interactions/hoverInteraction.ts

```diff
@@ -34,6 +34,7 @@ export module Interaction {
   export class Hover extends Interaction.AbstractInteraction {
     public _componentToListenTo: Hoverable;
     private _dispatcher: Dispatcher.Mouse;
+    private _touchDispatcher: Dispatcher.Touch;
```

Comment:
What do you mean?  If the touch events are thrown as mousemove then shouldnt mousemove catch it?